### PR TITLE
feat(cli): add `db get` subcommand

### LINKED ISSUE #1023: add `db get` subcommand
### Describe the feature

for running locally and troubleshooting, would be nice to have a `get` subcommand so we can `reth db get Headers --id 2000002` to quickly dump out one record. Related to #612 . 

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by gnapoli23**
Add `db get` subcommand to get info of a specific table.

This is a draft, so i would like to have some other guidance on how to use `seek_exact`, considered that it should be also used as method for `db list`, as specified in #612 

Fixes #1023

### CODE REVIEW

File: bin/reth/src/db/mod.rs [hunk 1]
```diff
@@ -232,6 +270,15 @@ impl<'a, DB: Database> DbTool<'a, DB> {
             .map_err(|e| eyre::eyre!(e))
     }
 
+    fn get<T:Table>(&mut self, idx: usize) {
```
Comment (gakonst) [inline bin/reth/src/db/mod.rs:273]:
```suggestion
    fn get<T:Table>(&mut self, idx: T::Key) -> T::VAlue {
```

This should be the table's key and you should just do `self.db.view(|tx| tx.get(key))` as far as i can tell

File: bin/reth/src/db/mod.rs [hunk 2]
```diff
@@ -178,6 +191,33 @@ impl Command {
...
+                        Ok::<(), eyre::Report>(())
+                    } else {
+                        // Get data
+                        // if let Some(_) = tool.get(args.idx)? {
+                        //     todo!();
+                        // } else {
+                        //     println!("Nothing found for index {idx} on table {table}", idx = args.idx, table = args.table);
+                        // }
+                        Ok(())
+                    }
```
Comment (gakonst) [inline bin/reth/src/db/mod.rs:218]: all of this should just be doing `get` and use a macro like the `View` command

**Review by gakonst — CHANGES_REQUESTED** [review]
lmk if you're gonna take this on in the next days, else would like to complete/close

**Comment by gnapoli23** [comment]
> lmk if you're gonna take this on in the next days, else would like to complete/close

Sorry I can't, currently OOO, have no access to my workstation.

**Comment by gakonst** [comment]
sounds good - closing for now to clean things up and we'll take over or we can revisit hwen ur back

**Pull request closed without merging**